container/root/tests/php-fpm/legacy.goss.yaml

```diff
@@ -53,3 +53,9 @@ package:
     installed: true
   php-yaml:
     installed: true
+
+file:
+  /etc/php/5.6/mods-available/newrelic.ini:
+    exists: true
+    contains:
+      - "${REPLACE_NEWRELIC_LICENSE}"
```

Comment:
Really basic check for now but should always be true during a build. The file path would only change at runtime.